pkg/scheduler/framework/runtime/framework.go

```diff
@@ -432,10 +432,9 @@ func (f *frameworkImpl) RunPreFilterExtensionAddPod(
 		}
 		status = f.runPreFilterExtensionAddPod(ctx, pl, state, podToSchedule, podToAdd, nodeInfo)
 		if !status.IsSuccess() {
-			msg := fmt.Sprintf("error while running AddPod for plugin %q while scheduling pod %q: %v",
-				pl.Name(), podToSchedule.Name, status.Message())
-			klog.Error(msg)
-			return framework.NewStatus(framework.Error, msg)
+			err := status.AsError()
+			klog.ErrorS(err, "Failed running AddPod extension for PreFilter plugin", "plugin", pl.Name(), "pod", klog.KObj(podToSchedule))
```

Comment:
Is that accurate to say `AddPod` is an extension?

Reply:
It's ok: we are not saying "extension point". OTH, it doesn't add any useful information over `Failed running AddPod for PreFilter plugin`, so I prefer that.